CMakeLists.txt

```diff
@@ -865,4 +865,18 @@ if(BUILD_TESTS)
       10000
       --use-websockets
   )
+
+  if(BUILD_END_TO_END_TESTS)
+    if("virtual" IN_LIST COMPILE_TARGETS)
+      add_test(NAME forum_app
+               COMMAND ${CMAKE_CURRENT_LIST_DIR}/samples/apps/forum/ci.sh
```

Comment:
Missing file from this PR.

Reply:
Yep, sorry. It's 2020, and not only does my car not drive itself, but my RCS can't even tell what I mean. I want my AI money back.